### DISCUSSION TITLE
How to update InstallLocation for an existing manifest file

### DISCUSSION THREAD

**Discussion by krishna-santosh** [microsoft/winget-create#335] [Q&A]
I submitted a **portable exe** to winget, but it is asking for admin privileges to install it via winget. I had to open powershell/cmd as admin to install it. 

I did some lookup and found that adding  `InstallLocation: "%APPDATA%"` to the manifest file can solve this issue. 
How do I add this to my existing manifest file ?

I tried `wingetcreate update <PackageID>` but it throws an error.

Current Manifest: [Windows-Spotlight](https://github.com/microsoft/winget-pkgs/tree/master/manifests/w/Windows-Spotlight/Windows-Spotlight/1.0.0.0)

**Comment by pzfeng** [comment]
谢谢您！邮件已收到！！！~_~

**Comment by Trenly** [comment]
Portable exe files make use of the `PortablePackageUserRoot` setting and the `PortablePackageMachineRoot` setting. By setting the install location, you would be manually overriding these for all users. More than likely, the issue you were experiencing around it asking for admin privileges is due to the fact that it is a portable exe file, and winget cannot create symlinks without admin mode. Some changes have been made to the CLI around this behavior, but I don't recall if it has been fully resolved yet.

**Reply by krishna-santosh** [reply]
Thank you @Trenly.

**Comment by Neil-in-Chicago** [comment]
wtf?

⁣Neil
--
***@***.***

The only intelligent way to discuss politics is on all fours.​
